hivemind_redis_database/__init__.py

```
@@ -9,48 +11,549 @@
+            res = []
+            if results and len(results) > 1:
+                for i in range(1, len(results), 2):
+                    client_key = results[i]
+                    if client_key.startswith(f"{self.index_prefix}:client:"):
+                        client_data = self.redis.get(client_key)
+                        if client_data:
+                            try:
+                                client = cast2client(client_data)
+                                if client.api_key != "revoked":
+                                    res.append(client)
+                            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)